### PR TITLE
FEATURE/TariikMVP/AssignDevelopersv2

### PR DESCRIPTION
**Pull request by TarikVelic**
--------------------------------------------------------------------
 MAIN
 * Added new style for assign developers
 * Added select which contains part time and full time options
 * Small changes in code --------------------------------------------------------------------
 OTHER
 * ///// --------------------------------------------------------------------
Screenshot_7
Screenshot_6

### CODE REVIEW

File: src/components/formFields/MultiSelect.js [hunk 1]
```diff
@@ -1,26 +1,108 @@
...
+    setEmploymentTypes(updatedEmploymentTypes);
   };
 
-  const value = item.value.map((i) =>
-    item.options.find((o) => o.value === i)
+  const optionElements = item.options.map((option) => (
+    <Option key={option.value} value={option.value}>
+      <div className="developer-option">
+        <Checkbox
+          onChange={(e) => e.stopPropagation()}
```
Comment (AntMarkoLovric) [inline src/components/formFields/MultiSelect.js]: move this outside of jsx

File: src/components/formFields/MultiSelect.js [hunk 2]
```diff
@@ -1,26 +1,108 @@
...
+          );
+          return (
+            <div key={developer.value} className="selected-developer">
+              {developer.label}
+              <div>
+                <Select {...employmentTypeSelectProps(developer)}>
+                  {employmentTypeOptionElements}
+                </Select>
+                <CloseOutlined
+                  onClick={() => onRemoveDeveloper(developer.value)}
```
Comment (AntMarkoLovric) [inline src/components/formFields/MultiSelect.js]: move this also

File: src/components/formFields/MultiSelect.js [hunk 3]
```diff
@@ -1,26 +1,108 @@
...
-        options={item.options}
-      />
+      <label htmlFor={item.id}>
+        {item.labelText}
+      </label>
+      <Select {...selectProps}>
+        {optionElements}
+      </Select>
+      <div className="selected-developers">
+        {item.value.map((developerId) => {
```
Comment (AntMarkoLovric) [inline src/components/formFields/MultiSelect.js]: move map outside of jsx

File: src/components/projects/components/AddProjectsModal/AddProjectsModal.js [hunk 4]
```diff
@@ -33,12 +36,36 @@ const AddProjectsModal = ({ handleClose, isOpen, isLoading, actions, employees }
...
+      const fullName = `${developer.firstName} ${developer.lastName}`;
+      return (
+        <div key={developer._id} className="selected-developer">
+          <span>{fullName}</span>
+          <Button
+            className= "button-x"
+            type="text"
+            size="small"
+            icon={<CloseOutlined />}
+            onClick={() => removeDeveloper(developer._id)}
```
Comment (AntMarkoLovric) [inline src/components/projects/components/AddProjectsModal/AddProjectsModal.js]: move this also

**Review by AntMarkoLovric — CHANGES_REQUESTED** [review]

**Review by becarselman — APPROVED** [review]

**Review by melcidar — APPROVED** [review]

**Review by AntMarkoLovric — APPROVED** [review]

**Pull request closed without merging**